src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -41,12 +41,14 @@ public Datastore() {
 
     public void savePuzzles() {
         Puzzle puzzle1 = new Puzzle(1, Puzzle.Puzzle_Type.TEXT, "At midday, I am the same size and shape of an elephant, but I do not weight the same as an elephant. At night, I am not seen at all. What am I?",
-            "ELEPHANTSSHADOW");
+            "(ELEPHANT'?S?)?\\s?SHADOW");
```

Comment:
is this the fixed reg-ex? does it accept 'the', 'an', etc?

Reply:
no this only accepts elephant('s) shadow